packages/front-end/components/Settings/UpgradeModal/index.tsx

```diff
@@ -253,6 +287,115 @@ export default function UpgradeModal({ close, source }: Props) {
     }
   };
 
+  const upgradeHeader = (
+    <>
+      <h3 className="mb-1">Upgrade to Pro</h3>
+      <p
+        className="mb-0"
+        style={{ color: "var(--color-text-mid", fontSize: "16px" }}
```

Comment:
@ahdriel We don't have any pre-built components that wrap Radix's `Text` components that abstract any of this custom styling, correct?

It feels like we may want to wrap the `Text` component and then define custom styles via the `as` property for things like labels, subheads, spans, etc, and probably the same for Radix's `Headings`.

Just wanted to double check this didn't already exist and that there wasn't an existing pattern I wasn't aware of to handle this.

Reply:
Yeah :/ I think maybe for the color and the font-size we need to have a global default via CSS using Radix variables? but it is a bigger sweeping change.

And yeah, we likely should create a `component/Radix/Text` that receives special props to define between low/mid/high text color (this is a concept we are adopting that does not exist in Radix).

Reply:
Thanks for following up and confirming - wanted to double check I wasn't missing any pre-existing solution for this. Thanks!

Reply:
This looks like a typo  - missing closing parentheses